### PR TITLE
Improve semantics

### PR DESCRIPTION
**Pull request by pchampin**
long overdue review of the semantics document

### CODE REVIEW

File: spec/semantics.html [hunk 1]
```diff
@@ -133,15 +155,15 @@ <h2>Abstract syntax</h2>
       </p>
       --> 
       In this section we define N3's abstract syntax. As N3's concrete concrete turtle-based syntax has proven 
-      to leave room for ambiguity - especially implicit quantification
-      has been interpreted contradictively by different reasoners in the past - we decided to introduce an abstract syntax in which aspects like quantification 
-      are stated explicitly. If the semantics of implicit quantification will need to be changed by the working group for any reason, this change will thus not affect the 
+      to leave room for ambiguity &ndash; especially implicit quantification
+      has been interpreted contradictively by different reasoners in the past &ndash;  we decided to introduce an abstract syntax in which aspects like quantification
```
Comment (TallTed) [inline spec/semantics.html:159]:
```suggestion
      to leave room for ambiguity &mdash; especially implicit quantification
      has been interpreted in contradicting ways by different reasoners in the past &mdash;  we decided to introduce an abstract syntax in which aspects like quantification 
```

File: spec/semantics.html [hunk 2]
```diff
@@ -133,15 +155,15 @@ <h2>Abstract syntax</h2>
...
+      has been interpreted contradictively by different reasoners in the past &ndash;  we decided to introduce an abstract syntax in which aspects like quantification 
+      are stated explicitly. If the semantics of implicit quantification needs to be changed by the community group for any reason, this change will thus not affect the 
       semantics, which is based on the abstract syntax, but only the mapping from concrete to abstract syntax. 
 
       <section class="informative">
         <h3>Motivation</h3>
-   <p>Notation3 supports implicit quantification. If blank nodes are used in triples, they stand for implicit existentially
-   quantified variables. In case universals (i.e., variables starting with a question mark in the concrete syntax) are used, they stand for implicit
+   <p>Notation3 supports implicit quantification. If blank nodes are used in triples, they stand for implicitly existentially
+   quantified variables. In case universals (i.e., variables starting with a question mark in the concrete syntax) are used, they stand for implicitly
```
Comment (TallTed) [inline spec/semantics.html:166]:
```suggestion
   <p>Notation3 supports implicit quantification. When blank nodes are used in triples, they implicitly stand for existentially
   quantified variables. When universals (i.e., variables starting with a question mark in the concrete syntax) are used, they implicitly stand for
```

File: spec/semantics.html [hunk 3]
```diff
@@ -133,15 +155,15 @@ <h2>Abstract syntax</h2>
...
    universally quantified variables. Implicit universal quantification is always global, that is, implicit universal quantification
    always spans over the full graph. Implicit existential quantification is always local, that is, it spans the local graph
    (e.g., if implicit existential quantification occurs in a graph term, it spans only that graph term).
```
Comment (TallTed) [inline spec/semantics.html:169]:
```suggestion
   universally quantified variables. Implicit universal quantification is always global; that is, implicit universal quantification
   always spans over the full graph. Implicit existential quantification is always local; that is, it spans the local graph
   (i.e., if implicit existential quantification occurs in a graph term, it spans only that graph term).
```

**Review by domel — APPROVED** [review]

File: spec/semantics.html [hunk 4]
```diff
@@ -247,37 +264,38 @@ <h3>Terms, Triples and Graphs</h3>
 </p>
 <div class=note>
   In order to improve readability, we will omit the inner 
-  graph brackets for all graph terms here and in the remainder of the document. That is, we will for example write 
-  <div style="text-indent:20px;"><small>$<\{\},\{x\},\{(x,\text{rdf:type}, \text{:Mortal} )\}>$</small></div>
-  <div>instead of</div>
-  <div style="text-indent:20px;"><small>$<(\{\},\{x\},\{(x,\text{rdf:type}, \text{:Mortal} )\})>$.</small></div>
+  graph round brackets for all graph terms here and in the remainder of the document. That is, we will for example write<br/>
```
Comment (TallTed) [inline spec/semantics.html:267]:
```suggestion
  graph parentheses ("<code>()</code>") for all graph terms here and in the remainder of the document. That is, for example, we will write<br/>
```

File: spec/semantics.html [hunk 5]
```diff
@@ -286,22 +304,22 @@ <h3>Terms, Triples and Graphs</h3>
...
         in connection with <a>logical predicates</a>.  
-        More concrete, that means that the N3-turtle triple 
-        <div style="text-indent:20px;"><code>:socrates :says {_:x a :Mortal}.  </code> </div> 
-        is interpreted 
-        as 
-        <div style="text-indent:20px;"><small> $(\{\},\{x\},\{(\text{:socrates}, \text{:says}, <\{\},\{\},\{(x,\text{rdf:type}, \text{:Mortal} )\}>) \})$</small></div> 
-         and <b>not</b> as
-         <div style="text-indent:20px;"> <small> $(\{\},\{\},\{(\text{:socrates}, \text{:says}, <\{\},\{x\},\{(x,\text{rdf:type}, \text{:Mortal} )\}>) \})$</small>. </div> 
-         A solution for that problem is currently being discussed. The difference, however, does not influence the formal semantics of the <i>abstract</i> syntax, as quantification is always explicit there.
+        More concrete, that means that the N3-turtle triple<br/>
```
Comment (TallTed) [inline spec/semantics.html:315]:
```suggestion
        More concretely, that means that the N3-turtle triple<br/> 
```

File: spec/semantics.html [hunk 6]
```diff
@@ -366,20 +384,20 @@ <h2>Variables</h2>
 
 
             <p class=note>Note that the definition makes a difference between <a>graphs</a> and <a>graph terms</a>.
-              A graph term is a <i>direct constituent</i> of the graph ("containing graph") it occurs in as a triple component. However, the direct constituents of the graph term's graph, and their direct constituents (and so on), are not direct constituents of the containing graph. Instead, they are <i>constituents</i> of the containing graph.
+              A graph term is a <a>direct constituent</a> of the graph ("containing graph") it occurs in as a triple component. However, the direct constituents of the graph term's graph, and their direct constituents (and so on), are not direct constituents of the containing graph. Instead, they are <a>constituents</a> of the containing graph.
```
Comment (TallTed) [inline spec/semantics.html:387]:
```suggestion
              A graph term is a <a>direct constituent</a> of the graph ("containing graph") in which it occurs as a triple component. However, the direct constituents of the graph term's graph, and their direct constituents (and so on), are not direct constituents of the containing graph. Rather, they are <a>constituents</a> of the containing graph.
```

File: spec/semantics.html [hunk 7]
```diff
@@ -440,39 +460,41 @@ <h2>Variables</h2>
...
+        <!--  <li> In <span class="asex"> $(\{\},\{\},\{(\text{x}, \text{:says}, &lt;\{\},\{x\},\{(x,\text{rdf:type}, \text{:Mortal} )\}&gt;) \})$</span> the
             $x$ is a free variable but it also occurs as a scoped variable in the quoted graph. -->
 </ol>
 
 </section>
 <section>
   <h2>Isomorphisms</h2>
-We would like to define our semantics in a way which assigns the <i>same meaning for structurally identical graphs</i>, i.e.,  which only differ in the naming of the variables they use. To be able to do so, we first introduce the concept of combined mappings which will be relevant to handle nested graphs with local 
+We would like to define our semantics in a way which assigns the <em>same meaning for structurally identical graphs</em>, i.e., graph which only differ in the naming of the variables they use. To be able to do so, we first introduce the concept of combined mappings which will be relevant to handle nested graphs with local 
 scoping.
```
Comment (TallTed) [inline spec/semantics.html:476]:
```suggestion
We would like to define our semantics in a way which assigns the <em>same
meaning for structurally identical graphs</em>, i.e., graphs which only
differ in the naming of the variables they use. To enable this, we first
introduce the concept of combined mappings which will be relevant to
handling nested graphs with local scoping.
```

File: spec/semantics.html [hunk 8]
```diff
@@ -626,12 +644,13 @@ <h2>Base Semantics</h2>
      
     <p>
       The basic interpretation maps graph terms to an isomorphic copy of themselves. As a consequence, two non-isomorphic graph terms cannot have the same meaning. For the abstract versions of the graph terms <code>{_:x :p :o}</code>, <code>{_:y :p :o}</code>
-      and <code>{_:x :p :o. _:y :p :o}</code> we thus get:
-      <ul>
-     <li> $Q_I($<code><({}, {y}, {(y, :p, :o)})></code>$)= Q_I($<code><{}, {x}, {(x, :p, :o)}></code>$)$
-      <li> $Q_I($<code><({}, {y}, {(y, :p, :o)})></code>$)\neq Q_I($<code><{}, {x, y}, {(x, :p, :o),(y, :p, :o) }></code>$)$
+      and <code>{_:x :p :o. _:y :p :o}</code> we thus get:</p>
```
Comment (TallTed) [inline spec/semantics.html:647]:
```suggestion
      and <code>{_:x :p :o. _:y :p :o}</code>, we thus get:</p>
```

File: spec/semantics.html [hunk 9]
```diff
@@ -626,12 +644,13 @@ <h2>Base Semantics</h2>
...
+    <ul>
+     <li> $Q_I(&lt;\{\}, \{y\}, \{(y, \text{:p}, \text{:o})\}&gt;)= Q_I(&lt;\{\}, \{x\}, \{(x, :p, :o)\}&gt;)$
+      <li> $Q_I(&lt;\{\}, \{y\}, \{(y, \text{:p}, \text{:o})\}&gt;) \neq Q_I(&lt;\{\}, \{x, y\}, \{(x, \text{:p}, \text{:o}), (y, \text{:p}, \text{:o})\}&gt;)$
     </ul>
 
+    <p>
       However, as we will see below, N3 provides mechanisms to <a>unquote</a>  graph terms (that is, to transform them into graphs) and to compare them based on the meaning of the formula they quote.
```
Comment (TallTed) [inline spec/semantics.html:654]:
```suggestion
      However, as we will see below, N3 provides mechanisms to <a>unquote</a>
      graph terms (that is, to transform them into graphs), and to compare them
      based on the meaning of the formulae they quote.
```

**Review by TallTed — CHANGES_REQUESTED** [review]

File: spec/semantics.html [hunk 10]
```diff
@@ -669,21 +684,21 @@ <h2>Base Semantics</h2>
...
           This choice was made to guarentee <a href="https://en.wikipedia.org/wiki/Opaque_context">referential opacity</a> 
           (see also <a href="https://www.w3.org/2021/12/rdf-star.html#ref-opacity">referential in RDF-star</a> ). 
           To illustrate this idea consider the <a href="https://iep.utm.edu/referential-opacity/">superman example</a> expressed in N3:
-          <br>
-          <div style="text-indent:20px;margin-top:0.2cm"><code>:superman owl:sameAs :clarkKent. </code></div> 
-            <div style="text-indent:20px;margin-bottom:0.2cm"><code>:loisLane :belives {:superman :can :fly}.</code> </div>  
-
+          </p>
+          <div class="codeblock"><code>:superman owl:sameAs :clarkKent.
+            :loisLane :belives {:superman :can :fly}.
```
Comment (william-vw) [inline spec/semantics.html]:
```suggestion
            :loisLane :believes {:superman :can :fly}.
```

File: spec/semantics.html [hunk 11]
```diff
@@ -695,16 +710,16 @@ <h2>Base Semantics</h2>
...
         In order to introduce the interpretation of graphs containing variables, we rely on a few additional definitions. To better understand the need for these, consider the 
-        following example:
+        following example:<br/>
 
         <p>
         <code>?x :says {?x :is :wise}.</code>
       </p>
 
-        which freely translates to "<i>everyone says about himself that he is wise</i>."  In abstract syntax, the formula would be expressed as:
+        which freely translates to <q>everyone says about themself that they are wise</q>.  In abstract syntax, the formula would be expressed as:
```
Comment (william-vw) [inline spec/semantics.html:717]:
```suggestion
        which freely translates to <q>everyone says about themselves that they are wise</q>.  In abstract syntax, the formula would be expressed as:
```

File: spec/semantics.html [hunk 12]
```diff
@@ -669,21 +684,21 @@ <h2>Base Semantics</h2>
 
         The definition implies that the empty graph is always true and that the empty list is always mapped to itself. 
         <div class="note">
+          <p>
           Note that the interpretation function $Q_I(x)$ maps ground graphs into the domain of discourse without interpreting their content seperately.
```
Comment (william-vw) [inline spec/semantics.html:686]:
```suggestion
          Note that the interpretation function $Q_I(x)$ maps ground graphs into the domain of discourse without interpreting their content separately. 
```

File: spec/semantics.html [hunk 13]
```diff
@@ -472,7 +472,7 @@ <h2>Variables</h2>
 </section>
 <section>
   <h2>Isomorphisms</h2>
-We would like to define our semantics in a way which assigns the <em>same meaning for structurally identical graphs</em>, i.e.,  which only differ in the naming of the variables they use. To be able to do so, we first introduce the concept of combined mappings which will be relevant to handle nested graphs with local 
+We would like to define our semantics in a way which assigns the <em>same meaning for structurally identical graphs</em>, i.e., graph which only differ in the naming of the variables they use. To be able to do so, we first introduce the concept of combined mappings which will be relevant to handle nested graphs with local
```
Comment (william-vw) [inline spec/semantics.html:475]:
```suggestion
We would like to define our semantics in a way which assigns the <em>same meaning for structurally identical graphs</em>, i.e., graphs which only differ in the naming of the variables they use. To be able to do so, we first introduce the concept of combined mappings which will be relevant to handle nested graphs with local 
```

**Review by william-vw — CHANGES_REQUESTED** [review]
See suggestions

**Review by william-vw — APPROVED** [review]

**Review by william-vw — CHANGES_REQUESTED** [review]
See suggestions

**Review by william-vw — APPROVED** [review]

**Pull request merged**